examples/gnrc_networking/Makefile

```diff
@@ -13,11 +13,8 @@ USEMODULE += netdev_default
 USEMODULE += auto_init_gnrc_netif
 # Activate ICMPv6 error messages
 USEMODULE += gnrc_icmpv6_error
-# Specify the mandatory networking module for a IPv6 routing node
-USEMODULE += gnrc_ipv6_router_default
-# Add a routing protocol
-USEMODULE += gnrc_rpl
-USEMODULE += auto_init_gnrc_rpl
```

Comment:
It still might be worth to have hosts in there as RPL leafs.

Reply:
Do you have any particular use case in mind?

Reply:
Well if you have a `gnrc_networking` router you might want it to also see the hosts (so they need to be a RPL leaf, i.e, keep the `gnrc_rpl` modules in regardless ;-))

Reply:
Also: IIRC this is currently our only real RPL example, so why remove it?

Reply:
Ah, locally I have `examples/gnrc_networking_rpl` but that's not upstream (anymore?).

Anyhow, I made it optional now.

Reply:
That was never upstream, it was part of #14623

But yea a dedicated RPL example would be better IMHO